doc/user_guide/dynamo0p3.rst

```diff
@@ -2014,6 +2014,25 @@ logic determined by their :ref:`access modes <dynamo0.3-kernel-valid-access>`.
   are updated (albeit in a different way due to different access to DoFs, see
   :ref:`dynamo0.3-api-meta-args` for more details).
 
+
+File Name Conventions
++++++++++++++++++++++
+LFRic development uses strict naming conventions related to kernels.
```

Comment:
Done - I admit I don't agree at all with our standard of marking headers, but done

As explanation: since all headers have the same #lines, you need to know the usage of ++/~~/-- to know what 'nesting level" (heading, subheading, subsubheading) something it. I find it visually much easier when looking at the text file to know what level you are on if the number of empty lines at least somewhat indicated the nesting level:
- 2 lines before one after heading = Top level
- 1 line before, one after = mid level
- 1 lines before, none after = bottom level

But that's just me, and admittedly it was never done consistently :)

Reply:
Yes, we're not consistent in this respect so I am going with what's already there. Perhaps we can raise this at the Telco tomorow.